playground/android/app/build.gradle

```diff
@@ -1,39 +1,32 @@
 apply plugin: "com.android.application"
 
-def enableHermes = false
 project.ext.react = [
-        root     : "../../../",
-        entryFile: "index.js",
-        bundleAssetName: "index.android.bundle",
-        bundleInAlpha: true,
-        bundleInBeta: true,
-        enableHermes: enableHermes,
-        hermesFlagsDebug:['-Xes6-proxy','-output-source-map'],
-        hermesFlagsRelease:['-output-source-map'],
-        hermesCommand: "../../../node_modules/hermes-engine/%OS-BIN%/hermesc",
+        root              : "../../../",
+        cliPath           : "node_modules/react-native/cli.js",
+        entryFile         : "index.js",
+        bundleAssetName   : "index.android.bundle",
+        bundleInAlpha     : true,
+        bundleInBeta      : true,
+        enableHermes      : true,
+        hermesFlagsDebug  : ['-Xes6-proxy', '-output-source-map'],
+        hermesFlagsRelease: ['-output-source-map'],
+        hermesCommand     : "../../../node_modules/hermes-engine/%OS-BIN%/hermesc",
 ]
 
-def jscFlavor = 'org.webkit:android-jsc:+'
-
 apply from: "../../../node_modules/react-native/react.gradle"
 
 android {
-    compileSdkVersion 29
-    ndkVersion "20.1.5948944"
-
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
+    compileSdkVersion 30
+    ndkVersion "23.0.7599858"
```

Comment:
we should stick to RN config https://github.com/facebook/react-native/blob/v0.65.1/gradle.properties#L7

Reply:
Done